### PR TITLE
#11627 - Duplicate case query is triggered twice on every search

### LINKED ISSUE #11627: Duplicate case query is triggered twice on every search
#### Steps to Reproduce

1.
2.

#### System Details

* Device:
* SORMAS version:
* Android version/Browser:
* Server URL:
* User Role:

### PR DESCRIPTION
**Pull request by sergiupacurariu**
Fixes #11627

### CODE REVIEW

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11673

File: sormas-ui/src/main/java/de/symeda/sormas/ui/caze/MergeCasesView.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ public MergeCasesView() {
 
 		addHeaderComponent(btnBack);
 
-		if (viewConfiguration.isFiltersApplied()) {
+		if (!viewConfiguration.isFiltersApplied()) {
```
Comment (leventegal-she) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/caze/MergeCasesView.java:100]: the is the same condition as in the `enter` method but it does the opposite si it is confusing. Sometimes we reload when the filters are not applied and sometimes we show a message to the user, it should be consistent.

Also by default this flag is false so it loads and shows the user a message the he should click on the Apply filters button to see the duplicates

**Review by leventegal-she — CHANGES_REQUESTED** [review]

File: sormas-ui/src/main/java/de/symeda/sormas/ui/caze/MergeCasesView.java (hunk 1 above)
Comment (sergiupacurariu) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/caze/MergeCasesView.java]: update the change

**Comment by sormas-vitagroup** [comment]
SonarCloud analysis: https://sonarcloud.io/dashboard?id=SORMAS-Project&pullRequest=11673

**Review by leventegal-she — APPROVED** [review]

**Pull request merged**